pkg/configuration/configuration.go

```diff
@@ -110,92 +110,92 @@ type ClusterNamespaces map[string]string
+	return loadClusterAccessDefinition(ksctlConfig, clusterName)
 }
 
-func loadClusterAccessDefinition(sandboxUserConfig SandboxUserConfig, clusterName string) (ClusterAccessDefinition, error) {
+func loadClusterAccessDefinition(ksctlConfig KsctlConfig, clusterName string) (ClusterAccessDefinition, error) {
 	// try converted to camel case if kebab case is provided
-	clusterDef, ok := sandboxUserConfig.ClusterAccessDefinitions[utils.KebabToCamelCase(clusterName)]
+	clusterDef, ok := ksctlConfig.ClusterAccessDefinitions[utils.KebabToCamelCase(clusterName)]
 	if !ok {
 		// if not found, then also try original format (to cover situation when camel case is used)
-		if clusterDef, ok = sandboxUserConfig.ClusterAccessDefinitions[clusterName]; !ok {
+		if clusterDef, ok = ksctlConfig.ClusterAccessDefinitions[clusterName]; !ok {
 			return ClusterAccessDefinition{}, fmt.Errorf("the provided cluster-name '%s' is not present in your ksctl.yaml file. The available cluster names are\n"+
 				"------------------------\n%s\n"+
-				"------------------------", clusterName, strings.Join(getAllClusterNames(sandboxUserConfig), "\n"))
+				"------------------------", clusterName, strings.Join(getAllClusterNames(ksctlConfig), "\n"))
 		}
 	}
 	if clusterDef.ClusterType == "" {
-		return ClusterAccessDefinition{}, fmt.Errorf("sandbox command failed: 'cluster type' is not set for cluster '%s'", clusterName)
+		return ClusterAccessDefinition{}, fmt.Errorf("ksctl command failed: 'cluster type' is not set for cluster '%s'", clusterName)
 	}
 	if clusterDef.ServerAPI == "" {
-		return ClusterAccessDefinition{}, fmt.Errorf("sandbox command failed: The server API is not set for the cluster %s", clusterName)
+		return ClusterAccessDefinition{}, fmt.Errorf("ksctl command failed: The server API is not set for the cluster %s", clusterName)
 	}
 	if clusterDef.ServerName == "" {
-		return ClusterAccessDefinition{}, fmt.Errorf("sandbox command failed: The server name is not set for the cluster %s", clusterName)
+		return ClusterAccessDefinition{}, fmt.Errorf("ksctl command failed: The server name is not set for the cluster %s", clusterName)
 	}
 	return clusterDef, nil
 }
 
-func getAllClusterNames(config SandboxUserConfig) []string {
+func getAllClusterNames(config KsctlConfig) []string {
 	var clusterNames []string
 	for clusterName := range config.ClusterAccessDefinitions {
 		clusterNames = append(clusterNames, utils.CamelCaseToKebabCase(clusterName))
 	}
 	return clusterNames
 }
 
-// ClusterConfig contains all parameters of a cluster loaded from SandboxUserConfig
-// plus all cluster names defined in the SandboxUserConfig
+// ClusterConfig contains all parameters of a cluster loaded from KsctlConfig
+// plus all cluster names defined in the KsctlConfig
 type ClusterConfig struct {
 	ClusterAccessDefinition
 	AllClusterNames  []string
 	ClusterName      string
 	Token            string
-	SandboxNamespace string
+	KubeSawNamespace string
```

Comment:
the var name remains a bit ambiguous IMO. Is it the `toolchain-host-operator` namespace (as we still call it now)? What about `Namespace` (very generic, I admit) or `OperatorNamespace` instead?

Reply:
yeah, it's a bit tricky since it can be either `toolchain-host-operator` or `toolchian-member-operator` ns. 
I was thinking about:
1. `KubeSawOperatorNamespace`
2. `KubeSawOpNamespace`
3. `KubeSawOperatorNs`
4. `KubeSawOpNs`

to make it clear which type of operator we are talking about. 
But yeah, `OperatorNamespace` could be fine.

Reply:
or `ksOperatorNamespace`

Reply:
having a comment on the field to explain that its value can be either `toolchain-host-operator` or `toolchain-member-operator` (depending on the type of cluster we're dealing with) would be nice, too.

Reply:
renamed to `operatorNamespace` in 3abbff6

Reply:
added comment: [7c66812](https://github.com/kubesaw/ksctl/pull/15/commits/7c66812810660cb4da5f3fa873dcc86429cad1ad)

Reply:
thanks @MatousJobanek!